unicodetools/data/ucd/dev/DerivedNormalizationProps.txt

```diff
@@ -9005,6 +9005,6077 @@ E01F0..E0FFF  ; NFKC_CF;                # Cn [3600] <reserved-E01F0>..<reserved-
 
 # ================================================
 
+# Derived Property:   NFKC_SimpleCasefold (NFKC_SCF)
+#   This property removes certain variations from characters: case, compatibility, and default-ignorables.
+#   It is used for loose matching and certain types of identifiers.
+#   It is constructed by applying NFKC, simple CaseFolding, and removal of Default_Ignorable_Code_Points.
+#   The process of applying these transformations is repeated until a stable result is produced.
+#   WARNING: Application to STRINGS must apply NFC after mapping each character, because characters may interact.
+#            For more information, see [https://www.unicode.org/reports/tr44/]
+# Omitted code points are unchanged by this mapping.
+# @missing: 0000..10FFFF; NFKC_SCF; <code point>
+
+#  All code points not explicitly listed for NFKC_SimpleCasefold
+#  have the value <codepoint>.
```

Comment:
Done; note that this required some extensions to the MakeUnicodeFiles parser.

Reply:
Thanks for fixing the `<code point>` in *public data files*.

It looks like the parser complications are from changing the *internal* MakeUnicodeFiles.txt from `<codepoint>` to `<code point>`. I don't mind that change, and it's nice for this option to be consistent with the public value, but having a space in the *internal* syntax may not be worth the complications of *that* parser.

But up to you, now that you have done the work...

Reply:
Note that the comment in the public data files directly comes from the internal value of the `skipValue` flag in MakeUnicodeFiles.txt:

https://github.com/unicode-org/unicodetools/blob/b7ccbe65c3a1ec9aadf416fa17ad6dc5fdbf1a9d/unicodetools/src/main/java/org/unicode/text/UCD/MakeUnicodeFiles.java#L1155<!--
-->[…]<!--
-->https://github.com/unicode-org/unicodetools/blob/b7ccbe65c3a1ec9aadf416fa17ad6dc5fdbf1a9d/unicodetools/src/main/java/org/unicode/text/UCD/MakeUnicodeFiles.java#L1166-L1167

Of course I could just have special-cased a replacement of `<codepoint>` to `<code point>`, but making MakeUnicodeFiles.java ever more magically hacky seems like a bad idea (if anything I would like to clean up its parser some more one of these days…).

Reply:
I suspected as much. My inclination would have been a small replacement hack :-)
Much less code than collecting multiple options pieces and unescaping them...

We could have also made the internal value more obviously different, e.g., `sameCodePoint` or similar.

But I am ok with it as is.

Reply:
Collecting multiple options pieces (https://github.com/unicode-org/unicodetools/pull/423/commits/d50130b7d2aeb08c5500bd06e8e36f2bb155afcc) was indeed quite dumb; note that I have since come to my senses and just written a regular expression to lex for whole pieces: https://github.com/unicode-org/unicodetools/pull/423/commits/26801e95cec854ace119a315000fd95952eac354.